cadc-access-control/src/main/java/ca/nrc/cadc/ac/client/Main.java

```diff
@@ -197,6 +203,12 @@ private static void usage()
         System.out.println("  --get --group=<uri>");
         System.out.println("  --delete --group=<uri>");
         System.out.println();
+        System.out.println("--list --group=<uri>");
```

Comment:
For the list option, the service should be specified with the parameter --resourceID, which will be in the form ivo://<authority>/gms

This should replace the --group=<uri> param for the --list command.